modules/eigenstratdatabasetools/snpcoverage/main.nf

```diff
@@ -0,0 +1,40 @@
+process EIGENSTRATDATABASETOOLS_SNPCOVERAGE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::eigenstratdatabasetools=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/eigenstratdatabasetools:1.1.0--hdfd78af_0':
+        'quay.io/biocontainers/eigenstratdatabasetools:1.1.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(geno), path(snp), path(ind), val(suffix)
```

Comment:
Suffix presumably would be same across all files, so should go in separate channel, this would make it slightly easier for a pipeline developer